main.go

```diff
@@ -2,16 +2,24 @@ package main
 
 import (
 	"os"
+	"path/filepath"
 
 	"github.com/containerd/containerd/pkg/seed"
+	"github.com/rancher/k3c/cmd/daemon"
 	"github.com/rancher/k3c/pkg/cli/app"
 	"github.com/sirupsen/logrus"
 )
 
 func main() {
 	seed.WithTimeAndRand()
-	app := app.New()
-	if err := app.Run(os.Args); err != nil {
-		logrus.Fatal(err)
+	_, self := filepath.Split(os.Args[0])
```

Comment:
This is the same as filepath.Base(os.Args[0]).

Reply:
Thanks, good catch.